ui/scss/component/_file-render.scss

```diff
@@ -411,111 +431,15 @@ $control-bar-icon-size: 0.8rem;
     pointer-events: none;
   }
 
-  // control-bar: time
   .vjs-control-bar {
-    .vjs-time-control {
-      line-height: $control-bar-height;
-      font-size: $control-bar-font-size;
-      display: flex;
-    }
-
-    .vjs-current-time {
-      padding-right: 0;
-      margin-right: 0.4rem;
-      @media (max-width: $breakpoint-small) {
-        margin-right: 0.2rem;
-      }
-    }
-
-    .vjs-time-divider {
-      min-width: unset;
-      padding: 0;
-      z-index: 0; // solves the grayed-out divider
```

Comment:
### Merge error?
The "grayed-out divider" seems to be lost in a merge?

Also, I know we talked about icons sizing before (e.g. "Auto" and other buttons back to being smaller) -- if that's part of the next version, that's fine.

![image](https://user-images.githubusercontent.com/64950861/150247420-ec4677ae-ddd8-4c24-8c7c-98574d56b44c.png)

Reply:
Yeah, they were supposed to be part of the next version, but I'm probably gonna fix this by the end of this week. The icons look pretty bad and displaced. It's probably just one random line that is destorting everything :| 

Fixed the divider, thx for the hint. I've already tried to fix this a few days ago, but I wasn't able to find the cause within a reasonable amount of time. 

![image](https://user-images.githubusercontent.com/94839278/150340499-51082c99-4fd9-4d56-b4e9-2dbdf75225f7.png)

Reply:
Fixed the icons. Might make some adjustments later to the volume slider.
![image](https://user-images.githubusercontent.com/94839278/150386904-41166f09-b411-4032-8754-20e4973bf183.png)